plugin/cache/README.md

```diff
@@ -19,7 +19,7 @@ cache [TTL] [ZONES...]
 ~~~
 
 * **TTL** max TTL in seconds. If not specified, the maximum TTL will be used, which is 3600 for
-    noerror responses and 1800 for denial of existence ones.
+    no error responses and 1800 for denial of existence ones.
```

Comment:
`noerror` is a DNS response code.  So the lack of the the space is intended, and OK.
```suggestion
    NOERROR responses and 1800 for denial of existence ones.
```

Reply:
Thanks for detail explain!